modules/nf-core/coptr/map/main.nf

```diff
@@ -0,0 +1,59 @@
+process COPTR_MAP {
```

Comment:
Can you please move all coptr/map code to a different PR and ping me there? Will be easier for me to review